frontend/device/android/powerd.lua

```diff
@@ -69,6 +70,11 @@ function AndroidPowerD:turnOffFrontlightHW()
         return
     end
     android.setScreenBrightness(self.fl_min)
+
+    if device_model == "kon_tiki2" then -- for Onyx Boox Kon-Tiki 2 only
```

Comment:
Oh, I thought that was a codename (like how Kobo H2O = Dahlia, etc.). Still, it's useful to mention that it's Onyx. ;-)

Reply:
Yep) Onyx Boox always has interesting names for models in the Russian market. Usually they are after someone special like authors/scientists/discoverers (Akunin book, Lomonosov, Darwin, Livingstone, Magellan, etc), or some characters (Monte Cristo, Faust, etc). Don't know why they opted for generic Nova, Note, Lumi for international market though.